pkg/model/serializer.go

```diff
@@ -284,6 +284,24 @@ func (sync Sync) MarshalYAML() (interface{}, error) {
 	return syncRaw(sync), nil
 }
 
+// UnmarshalYAML Implements the Unmarshaler interface of the yaml pkg.
+func (buildInfo *DevBuildInfo) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	var rawString string
+	err := unmarshal(&rawString)
+	if err == nil {
+		buildInfo.Name = rawString
+		return nil
+	}
+	var rawBuildInfo BuildInfo
+	err = unmarshal(&rawBuildInfo)
+	if err != nil {
+		return err
+	}
+	oktetoLog.Yellow("The extended syntax is deprecated for dev. You can still build your images setting you build section on your manifes")
```

Comment:
```suggestion
	oktetoLog.Yellow("The `image` extended syntax is deprecated. Define the images you want to build in the 'build' section of your manifest")
```
we need a link to the docs too